app/pages/complete-order/contextCreator.js

```diff
@@ -0,0 +1,13 @@
+import withFundingManifest from './withFundingManifest.json';
+import { baseUrl } from '../../config';
+
+export const getContext = ({ orderId, orderDescription, fundingSource }) => {
+  const manifest = fundingSource ? withFundingManifest : undefined;

```

Comment:
sweet so this is getting it ready for the withoutFundingSource manifest to be provided later?

Reply:
yeah exactly thought this was a neat solution for the 2 manifests as me and sam R are pairing on them both